src/appleseed/renderer/kernel/lighting/pt/ptlightingengine.cpp

```diff
@@ -115,6 +118,9 @@ namespace
               , m_enable_ibl(params.get_optional<bool>("enable_ibl", true))
               , m_enable_caustics(params.get_optional<bool>("enable_caustics", false))
               , m_max_path_length(nz(params.get_optional<size_t>("max_path_length", 0)))
+              , m_max_specular_bounces(n_minus_o(params.get_optional<int>("max_specular_bounces", -1)))
+              , m_max_glossy_bounces(n_minus_o(params.get_optional<int>("max_glossy_bounces", -1)))
```

Comment:
What about we rename `fz()` to `fixup_path_length()` and `n_minus_o()` to `fixup_bounces()`?